docs/mddocs/Quickstart/llama_cpp_quickstart.md

```diff
@@ -296,6 +296,18 @@ Log end
 
 ### Troubleshooting
 
+#### Unable to run the initialization script
+If you are unable to run `init-llama-cpp.bat`, please make sure you have installed `ipex-llm[cpp]` in your conda environment. If you have installed it, please check if you have activated the correct conda environment. Also, if you are using Windows, please make sure you have run the script with administrator privilege in prompt terminal.
+
+#### `DeviceList is empty. -30 (PI_ERROR_INVALID_VALUE)` error
+On Linux, this error happens when devices starting with `[ext_oneapi_level_zero]` are not found. Please make sure you have installed level-zero, and have sourced `/opt/intel/oneapi/setvars.sh` before running the command.
+
+#### `Prompt is too long` error
+If you encounter `main: prompt is too long (xxx tokens, max xxx)`, please either reduce the length of the prompt or increase the `-c` parameter to set a larger size of the prompt context.
```

Comment:
If you encounter `main: prompt is too long (xxx tokens, max xxx)`, please increase the `-c` parameter to set a larger size of context.